types/netmap/netmap_test.go

```diff
@@ -6,17 +6,22 @@ package netmap
 
 import (
 	"encoding/hex"
+	"fmt"
+	"strings"
 	"testing"
 
+	"go4.org/mem"
 	"inet.af/netaddr"
 	"tailscale.com/tailcfg"
+	"tailscale.com/types/key"
 )
 
-func testNodeKey(b byte) (ret tailcfg.NodeKey) {
-	for i := range ret {
-		ret[i] = b
+func testNodeKey(b byte) key.NodePublic {
+	k, err := key.ParseNodePublicUntyped(mem.S(strings.Repeat(fmt.Sprintf("%02x", b), 32)))
```

Comment:
This is a long line, but what it accomplishes is very simple. Add a short doc comment or example to the function so the reader doesn't have to puzzle it out?